DigitalLearningSolutions.Data/DataServices/UserDataService/UserDataService.cs

```diff
@@ -87,6 +87,7 @@ string emailAddress
 
         int GetDelegateCountWithAnswerForPrompt(int centreId, int promptNumber);
         void DeleteAllAnswersForPrompt(int centreId, int promptNumber);
+        void ReactivateDelegateUser(int delegateId);

```

Comment:
Minor, but could we rename this "ActivateDelegateUser" as it is slightly more general.